src/components/FormRenameStep.vue

```diff
@@ -44,20 +44,22 @@ import renameSchema from '@/assets/schemas/rename-step__schema.json';
 import WidgetInputText from './WidgetInputText.vue';
 import WidgetAutocomplete from '@/components/WidgetAutocomplete.vue';
 import { Getter, Mutation, State } from 'vuex-class';
+import { StepFormComponent } from '@/components/formlib';
 
 interface RenameStepConf {
   oldname: string;
   newname: string;
 }
 
-@Component({
+@StepFormComponent({
+  vqbstep: 'rename',
   name: 'form-rename-step',
   components: {
     WidgetAutocomplete,
     WidgetInputText,
   },
 })
-export default class FormRenameStep extends Mixins(FormMixin) {
+export default class RenameStepForm extends Mixins(FormMixin) {
```

Comment:
Any particular reason to have change the class name here ?

Reply:
Nope, only that the latter seems more correct :)